src/components/calendar/top-part-of-calendar.tsx

```diff
@@ -36,6 +36,14 @@ export const TopPartOfCalendar: React.FC<TopPartOfCalendarProps> = ({
       >
         {value}
       </text>
+      <line
+        x1={x1Line}
+        y1={y2Line}
+        x2={x1Line + 12*30}
```

Comment:
can you remove the hardcoded `30` from here and we start using column width variable that we are setting from the parent component?

Reply:
We have discussed it offline, and we have targeted it in the issue https://github.com/coloredcow-admin/pmiic/issues/404 to make it dynamic.